app/bundles/ApiBundle/Form/Type/ConfigType.php

```diff
@@ -28,7 +30,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder->add(
             'api_enabled',
-            'yesno_button_group',
+            YesNoButtonGroupType::class,
```

Comment:
We are only supposed to change form types that belong to the current bundle/plugin (bundle that is being currently refactored) or Symfony core types.
This line should be changed as a part of CoreBundle refactoring, because this form type is defined in the CodeBundle.
Sorry I didn't mention it when I was writing the first review comment.

Reply:
Good to know 😄 I'll switch it back.

Reply:
@mtshaw3 I thing you can leave it here. Just don't replace it in future PRs. We worry it would cause merge conflicts. But in this case git should see that the lines are the same. Hopefully. If not, the conflict should be simple to resolve.